kv/memberlist/memberlist_client.go

```diff
@@ -140,6 +140,9 @@ type KVConfig struct {
 	AdvertiseAddr string `yaml:"advertise_addr"`
 	AdvertisePort int    `yaml:"advertise_port"`
 
+	ClusterLabel                     string `yaml:"cluster_label" category:"experimental"`
+	ClusterLabelVerificationDisabled bool   `yaml:"cluster_label_verification_disabled" category:"experimental"`
```

Comment:
Shall we call this `ClusterLabelVerificationDisabled` and make it default to `true`? Most of our other options are `-enabled`, not `-disabled`. Reverse logic is confusing to users.